docs/guides-stm32/3pln-stock-wiring-integration.md

```diff
@@ -20,40 +20,44 @@ SSR   | Solid-State Relay | Relay placed in-line with the boiler heaters for con
 With this doc update the PCB and component installs have been synchronized to both utilize 3PLN, harmonizing the install process. Although the images shown in this guide may use the component build the install principles and machine connection points are the same for the PCB when using the stock wire harness.
```

Comment:
I know it's not in your diff, but I suggest removing the "With this doc update..." sentence, as it reads more like a release note.